### PR TITLE
[CS-2400] Add discord link in settings panel

### PR DESCRIPTION
**Pull request by topwebtek7**
### Description

- [x] Completes #CS-2400

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

image

### CODE REVIEW

**Review by lukemelia — APPROVED** [review]
Looks good. +1 for eliminating a rainbow reference too.

File: cardstack/src/constants.ts [hunk 1]
```diff
@@ -3,3 +3,11 @@ export const TRANSACTION_PAGE_SIZE = 100; // Temp increase the page size
...
 export const SEND_TRANSACTION_ERROR_MESSAGE =
   'An error has occurred, could not send. If this persists, please contact support@cardstack.com';
+export const SettingsExternalURLs = {
+  cardstackHomepage: 'https://cardstack.com/',
+  review:
+    'itms-apps://itunes.apple.com/us/app/appName/id1457119021?mt=8&action=write-review',
+  twitterDeepLink: 'twitter://user?screen_name=cardstack',
+  twitterWebUrl: 'https://twitter.com/cardstack',
+  discordInviteLink: 'https://discord.gg/NYtJY5Fx',
+};
```
Comment (danibonilha) [inline cardstack/src/constants.ts:15]: I think we are starting to mix contexts by extracting scoped things into this constants file, this is like a strings object which is only used in the settings section, so why does it need to live in the global constants files ?

**Review by danibonilha — APPROVED** [review]
looks good, just one small comment

File: cardstack/src/constants.ts (hunk 1 above)
Comment (topwebtek7) [reply cardstack/src/constants.ts:15]: it can be used in our files as well and also easy to find & update later.

**Pull request merged**